.github/ISSUE_TEMPLATE/prepare_minor_release.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Minor release tracking issue
+title: Prepare for v<release-tag>
+---
+
+Please see this documentation for more details baout release process: https://github.com/metal3-io/metal3-docs/blob/main/processes/releasing.md
+
+
+**Note**:
+* The following is based on the v1.7 minor release. Modify according to the tracked minor release.
+
+## Tasks
+
+* [ ] Add/edit JJBs to acommodate new build jobs and PR jobs for new release branches. [Prior art](https://gerrit.nordix.org/c/infra/cicd/+/20875)
+* [ ] Add Prow jobs for new release branches in Project-infra repo [Prior art](https://github.com/metal3-io/project-infra/pull/697)
+* [ ] Prepare dev-env to acommodate new release related changes and configurations [Prior art](https://github.com/metal3-io/metal3-dev-env/pull/1381)
+* [ ] Update Metal3 book. [Prior art](https://github.com/metal3-io/metal3-docs/pull/413)
+* [ ] Release Ironic, also check new image is created in quay. [Prior art](https://github.com/metal3-io/ironic-image/tags)
```

Comment:
Ironic-image release process is not part of CAPM3 minor release process. It is released separately when upstream Ironic has bugfix branch available. Maybe mark it as optional ?

Reply:
fixed.